### DISCUSSION TITLE
cannot import name 'ReflectedIndex' from 'sqlalchemy.engine.interfaces' (/home/ec2-user/anaconda3/envs/pytorch_p310/lib/python3.10/site-packages/sqlalchemy/engine/interfaces.py)

### DISCUSSION THREAD

**Discussion by Yeementh24** [sqlalchemy/sqlalchemy#10538] [Usage Questions]
getting the error while importing llama_index

**Comment by CaselIT** [comment]
Hi,

Ensure you have the required version on sqlalchemy installed.

If a 3rd party library is causing that import issue please report it to them
